hack/deploy-custom-nto.sh

```diff
@@ -31,7 +32,14 @@ cvo_scale() {
 
 nto_deploy_custom() {
   oc project openshift-cluster-node-tuning-operator
-  oc patch deploy cluster-node-tuning-operator -p '{"spec":{"template":{"spec":{"containers":[{"env":[{"name":"CLUSTER_NODE_TUNED_IMAGE","value":"'$IMAGE'"}],"name":"cluster-node-tuning-operator"}]}}}}'
+  cd "${WORKDIR}" || { echo "failed to change dir to ${WORKDIR}"; exit; }
+  for file in ${WORKDIR}/manifests/*; do
+    # We will skip the ibm specific deployment. 
+    if [[ "$file" != ${WORKDIR}/manifests/50-operator-ibm* && "$file" == *.yaml ]]; then
```

Comment:
Why don't we just list `*.yaml` files instead of the check for a `*.yaml` file here?